release.md

```diff
@@ -0,0 +1,25 @@
+# Version 1.1 2017-11-28
+
+## Fixed Issues
+The following issues are fixed in this release:
+* [Use of moved executor in futures.hpp:1234](https://github.com/stlab/libraries/issues/94)
+* [Stack overflow with exception handling in futures](https://github.com/stlab/libraries/issues/93)
+* [(non-default) Executors and make_ready_future](https://github.com/stlab/libraries/issues/85)
+* [stlab::async will not accept non-const reference parameters](https://github.com/stlab/libraries/issues/80)
+* [when_all does not support continuing futures that return void](https://github.com/stlab/libraries/issues/79)
+* [Support move-only tasks](https://github.com/stlab/libraries/issues/69)
+* [task_system and notification_queue doesn't work with move only lambda](https://github.com/stlab/libraries/issues/59)
+* [channel TODO get process state under mutex?](https://github.com/stlab/libraries/issues/26)
+* [future TODO get_try()](https://github.com/stlab/libraries/issues/18)
+
+## New Functionality
```

Comment:
We also added stlab::serial_queue, stlab::scope, and stlab::task in the 1.1 release.

Reply:
Also, stlab::copy_on_write, stlab::for_each_argument, and the suite of archetype test classes (regular, move-only, etc.)